Enterprise-CMCS/macpro-mdct-mcr
# Add (optional) styling to fields that are not required

## Add `(optional)` styling to fields that are not required
[jira ticket](https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2237)

## Summary
Added '(optional)' text to fields. New attribute `styleAsOptional` can be added to mcpar.json to add `(optional)` at the end of the title for a field.

### How to test
Add  `"styleAsOptional": true` in the `props:` object of any field except dynamic fields (should support text, number, radio/checkbox, date, dropdown, single checkbox, etc.)` and check in the browser that the targeted field's title now has `(optional)` at the end of it. This has nothing to do with the field's validation, as that is controlled by `validation` type.

### Important updates


### Author checklist

- [ x] I have performed a self-review of my code
- [ x] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
